SeleniumSolarTechnology/SolarTechnology/Tests/UITestFixture.cs

```diff
@@ -0,0 +1,36 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium.Support.UI;
+
+namespace SolarTechnology.Tests
+{
+    public class UITestFixture
+    {
+        [SetUp]
+        public void Setup()
+        {
+            var options = new ChromeOptions();
+            options.AddArgument("window-size=1400,1200"); // Set desired resolution
+            _driver = new ChromeDriver(options);
+            _driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+            _js = (IJavaScriptExecutor)_driver;
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            _driver.Quit();
+        }
+
+
+        public IWebDriver _driver;
+        public IJavaScriptExecutor _js;
```

Comment:
these 2 variables should be at the top of the file, before [Setup]

Reply:
Thanks, done.